src/elemeno_ai_sdk/ml/autofeatures/client.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict, Optional
+
+import aiohttp
+
+from elemeno_ai_sdk.ml.mlhub_client import MLHubRemote
+from elemeno_ai_sdk.utils import mlhub_auth
+
+
+class AutoFeaturesClient(MLHubRemote):
+    auto_features_ref = "YWktZmVhdHVyZS1lbmdpbmVlcmluZw=="
```

Comment:
Set `auto_features_ref` in UPPER CASE, just to follow python style for constant variable definition